pkg/networks/usernet/client.go

```diff
@@ -140,7 +141,7 @@ func create(sock string, subnet net.IP, base string) *Client {
 	client := &http.Client{
 		Transport: &http.Transport{
 			DialContext: func(context.Context, string, string) (net.Conn, error) {
-				return net.Dial("unix", sock)
+				return netutil.DialUnix("unix", sock)
```

Comment:
In this case we already dropped the context. Maybe we should also drop it in other places, but it is not required to add a helper and can be done later.